src/raftstore/store/peer.rs

```diff
@@ -369,8 +369,39 @@ impl Peer {
         where T: Transport
     {
         for msg in msgs {
-            if msg.get_msg_type() == MessageType::MsgRequestVote {
-                PEER_RAFT_READY_COUNTER_VEC.with_label_values(&["request_vote"]).inc();
+            match msg.get_msg_type() {
```

Comment:
I worry this may reduce the performance with sending message.  @overvenus 
We may cache the metric and add in one batch.